src/event-db/migrations/V8__catalyst_automation.sql

```diff
@@ -28,18 +28,49 @@ COMMENT ON COLUMN voting_node.netkey IS
 Used as the node p2p topology key.';
 
 
--- Voting Nodes Table - Defines nodes in the network
--- This table is looked up by hostname
+-- Tally Committee Table - Stores data about the tally committee per voting event
+-- This table is looked up by event
 CREATE TABLE tally_committee (
     row_id SERIAL PRIMARY KEY,
 
     event INTEGER NOT NULL UNIQUE,
 
-    -- Other data goes here.
+    pubkey TEXT NOT NULL,
+    seckey TEXT NOT NULL,
+    comm_pk TEXT NOT NULL,
+    comm_sk TEXT NOT NULL,
 
     FOREIGN KEY(event) REFERENCES event(row_id)
 );
 
 COMMENT ON TABLE tally_committee IS 'Table for storing data about the tally committee per voting event.';
 COMMENT ON COLUMN tally_committee.row_id IS 'Unique ID for this committee member for this event.';
 COMMENT ON COLUMN tally_committee.event  IS 'The event this committee member is for.';
+COMMENT ON COLUMN tally_committee.pubkey  IS 'Public key for the committee wallet.';
+COMMENT ON COLUMN tally_committee.seckey  IS '';
+COMMENT ON COLUMN tally_committee.comm_pk  IS '';
+COMMENT ON COLUMN tally_committee.comm_sk  IS '';
+
+
+-- Committee Member Table - Stores data about the tally committee members
+-- This table is looked up by committee
+CREATE TABLE committee_member (
+    row_id SERIAL PRIMARY KEY,
+
+    committee INTEGER NOT NULL,
+
+    pubkey TEXT NOT NULL,
+    seckey TEXT NOT NULL,
+    member_pk TEXT NOT NULL,
+    member_sk TEXT NOT NULL,
+
+    FOREIGN KEY(committee) REFERENCES tally_committee(row_id)
+);
+
+COMMENT ON TABLE committee_member IS 'Table for storing data about the tally committee members.';
+COMMENT ON COLUMN committee_member.row_id IS 'Unique ID for this committee member for this event.';
+COMMENT ON COLUMN committee_member.committee IS 'The committe this member belongs to.';
+COMMENT ON COLUMN committee_member.pubkey  IS 'Public key for the member wallet.';
+COMMENT ON COLUMN committee_member.seckey  IS 'Secret key for the member wallet.';
+COMMENT ON COLUMN committee_member.member_pk  IS 'Committee member public key';
+COMMENT ON COLUMN committee_member.member_sk  IS 'Committee member secret key';
```

Comment:
Are the secret keys being stored encrypted in the DB?
We need to be able to safely publish the DB data, so any "Secret" keys we store need to be encrypted.
We need a `SECRET_SECRET` env var to control the "Secret" encryption key.
We should use this method:
https://cryptography.io/en/latest/fernet/#using-passwords-with-fernet

You need to derive the key on every encryption and decryption, which needs the salt.
Salt needs to be random on every encryption.
Save the salt with the encrypted data in the DB.
For example,  the salt is 16 bytes binary.
Get the encrypted binary data, pre-pend it with the 16 byte salt, base 64 the result and store that base 64 string as the encrypted secret.

You should just make a function which does this, and constrain the encryption/decryption to the DB layer on read and write.

Reply:
Got it, I've added the functions that do encrypt/decrypt for storage.